libsession/src/main/java/org/session/libsession/messaging/sending_receiving/ReceivedMessageHandler.kt

```diff
@@ -160,6 +172,13 @@ fun MessageReceiver.handleUnsendRequest(message: UnsendRequest) {
         SSKEnvironment.shared.notificationManager.updateNotification(context)
     }
 }
+
+fun handleMessageRequestResponse(message: MessageRequestResponse) {
```

Comment:
I think we also need to handle the new configuration fields to sync approved / blocked contacts from our own device just above ^

Reply:
We handle that in `storage.addContacts(..)` call above.